discover/deployment/articles-dxp/01-installing-liferay/07-installing-liferay-on-tcserver.markdown

```diff
@@ -31,35 +26,37 @@ $$$
 
 ## Installing @product@ Dependencies [](id=installing-liferay-dependencies)
 
-@product@ depends on some additional JARs that aren't included with tc Server by
-default. There are even more JARs that you'd find in a @product@ bundle
-that are not required but can be useful. If you don't have a @product@
-bundle, you can download the required JARs from third parties, as
-described below.
+@product depends on some additional JARs that are not included with tc Server by
+@default. There are even more JARs found in a product bundle that are not
+@required but can be useful. If you don't have a product bundle, you can
+@download the required JARs from third parties, as described below.
 
 +$$$
 
-**Note:** Many required and useful JARs are pre-installed when you build @product@ from the source code or [download a @product@ bundle](https://www.liferay.com/downloads/). If you want to acquire all of the
-JARs that ship with a @product@ bundle quickly, using one of these sources might
-save you time.
+**Note: Many required and useful JARs are pre-installed when you build @product@
+**from the source code or [download a @product@
+**bundle](https://web.liferay.com/group/customer/dxp/downloads/digital-
+**enterprise). If you want to acquire all of the JARs that ship with a @product@
+**bundle quickly, using one of these sources might save you time.
 
```

Comment:
Same thing here. Two stars (\*\*) in Markdown make something bold. Something weird happen with your editor when you reformatted the paragraph, I think.